cloud_automation/java/dataflow/README.md

```diff
@@ -0,0 +1,13 @@
+# CSVImport Template 
+
+The CSVImport dataflow template creates dataflow jobs to ingest CSV data from
+GCS to Cloud Bigtable.
+
+The template is stored at  gs://datcom-dataflow-templates/templates
+
+To deploy changes to the template run:
+
+```
+  mvn compile exec:java -Dexec.mainClass=org.datacommons.dataflow.CsvImport -Dexec.args="--runner=DataflowRunner --project=google.com:datcom-store-dev --stagingLocation=gs://datcom-dataflow-templates/staging --templateLocation=gs://datcom-dataflow-templates/templates/csv_to_bt"
+
```

Comment:
What does --stagingLocation mean?

Can we call it "csv_to_bt_staging"? (just to make it clear)

Reply:
staginglocation is the gcs directory where additional files required for running this template are stored. This is part of the dataflow custom template deployer and not out template code. I think it is best to leave it as-is, since it would require changing the dataflow template deployer,

Reply:
Oh I see. I misunderstood it to mean an equivalent of "staging instance".